src/pages/index.js

```diff
@@ -83,30 +166,55 @@ function createCard(data) {
 
 function handleAddCardFormSubmit(values) {
   const name = values.title;
+  // console.log(values);
   const link = values.URL;
   // generateCard({ name, link }, cardsWrap);
-  const cardEl = createCard({ name, link });
-  cardSection.addItem(cardEl);
+  api.addCards({ title: values.title, url: values.URL }).then((res) => {
+    newCardPopup.setButtonText(true, "Saving...");
```

Comment:
`    newCardPopup.setButtonText(true, "Saving...");
`

You need to set the loading status **before** the request (not in `then`)